From 041b5baea5a4efceff33df2f316646b47a1cd7c0 Mon Sep 17 00:00:00 2001
From: Sam James <11667869+thesamesam@users.noreply.github.com>
Date: Sat, 14 Aug 2021 17:29:24 +0100
Subject: [PATCH] src/fspp/fuse/Fuse.h: add <memory.h> include for
 std::shared_ptr (#395)

Needed to fix build with Boost 1.77 (some indirect inclusion
got lost).

Signed-off-by: Sam James <sam@gentoo.org>
---
 src/fspp/fuse/Fuse.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/fspp/fuse/Fuse.h b/src/fspp/fuse/Fuse.h
index 2dd3c2d4d..3841e38a4 100644
--- a/src/fspp/fuse/Fuse.h
+++ b/src/fspp/fuse/Fuse.h
@@ -11,6 +11,7 @@
 #include <boost/optional.hpp>
 #include <cpp-utils/macros.h>
 #include <atomic>
+#include <memory>
 #include "stat_compatibility.h"
 #include <fspp/fs_interface/Context.h>
 

